SYCL/Plugin/sycl-ls-gpu-default-any.cpp

```diff
@@ -3,9 +3,9 @@
 // RUN: env --unset=SYCL_DEVICE_FILTER sycl-ls --verbose >%t.default.out
 // RUN: FileCheck %s --check-prefixes=CHECK-GPU-BUILTIN,CHECK-GPU-CUSTOM --input-file %t.default.out
 
-// CHECK-GPU-BUILTIN: gpu_selector(){{.*}}gpu, {{.*}}Level-Zero
+// CHECK-GPU-BUILTIN: gpu_selector(){{.*}}gpu, {{.*}}{{Level-Zero|CUDA}}
```

Comment:
I wonder why two `CHECK` prefixes instead of using `CHECK-DAG`... Unrelated to this PR though.